lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.ZoneHVACComponent.rb

```diff
@@ -64,8 +64,7 @@ def zone_hvac_component_apply_prm_baseline_fan_power(zone_hvac_component)
       dsn_air_flow_cfm = OpenStudio.convert(dsn_air_flow_m3_per_s, 'm^3/s', 'cfm').get
 
       # Determine allowable fan BHP and power
-      # TODO: adjustments for single zone system, exisiting method is design around air loops
-      allowable_fan_bhp = 0.00094 * dsn_air_flow_cfm
+      allowable_fan_bhp = 0.00094 * dsn_air_flow_cfm + thermal_zone_get_fan_power_limitations(zone_hvac.thermalZone.get, false)
```

Comment:
Add credit/deductions for zonal system. This is only used for system 12, and 13.